assets/style.sass

```diff
@@ -104,7 +107,7 @@ section
     text-align: left
 
 .block
-  page-break-inside: avoid
+  break-inside: avoid
```

Comment:
What's the skinny in preferring `break-inside` over `page-break-inside`? I just skimmed the spec, I am guessing you like it better because it is less specific to print media?